.env.example

```
@@ -1,45 +1,11 @@
-FRONTEND_URL="http://localhost:1420"
-DOMAIN="localhost"
-COOKIE_NAME="incmix_session_dev"
-
-
-# Database
+NODE_ENV="test"
 DATABASE_URL="postgresql://postgres:password@localhost:54321/incmix"
+REDIS_URL="redis://:redis_password@localhost:6379"
 
+FRONTEND_URL="http://localhost:1420"
+DATABASE_URL="postgresql://postgres:password@localhost:54321/incmix"
```

dotenv-linter (3.3.0)

[warning] [DuplicatedKey] The DATABASE_URL key is duplicated

(DuplicatedKey)